db/hints/host_filter.cc

```diff
@@ -100,16 +105,17 @@ std::string_view host_filter::enabled_kind_to_string(host_filter::enabled_kind e
     throw std::logic_error("Uncovered variant of enabled_kind");
 }
 
-std::ostream& operator<<(std::ostream& os, const host_filter& f) {
-    fmt::print(os, "host_filter{{enabled_kind={}",
+}
+}
+
+auto fmt::formatter<db::hints::host_filter>::format(const db::hints::host_filter& f, fmt::format_context& ctx) const
+    -> decltype(ctx.out()) {
```

Comment:
broken lines are indented with double indent, so they stand out compared to the function body below

Reply:
ack. reindented.